packages/ui-time-select/src/TimeSelect/index.tsx

```diff
@@ -330,17 +330,20 @@ class TimeSelect extends Component<TimeSelectProps, TimeSelectState> {
   handleInputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
     const value = event.target.value
     const newOptions = this.filterOptions(value)
-
     this.setState({
       inputValue: value,
       filteredOptions: newOptions,
       highlightedOptionId: newOptions.length > 0 ? newOptions[0].id : undefined,
       isShowingOptions: true
     })
-
     if (!this.state.isShowingOptions) {
       this.props.onShowOptions?.(event)
     }
+    this.props.onInputChange?.(
+      event,
+      value,
```

Comment:
Somehow we need to pass the date/time value as well, not just the text value. It will be needed, in case someone wants to use the new prop in a controlled mode.

Reply:
I've added a new field thats an ISO datetime string